exercises/concept/temperature/.docs/hints.md

```diff
@@ -0,0 +1,13 @@
+# General
+
+Temperature in degrees Celsius is defined as: T(°F) = T(°C) × 1.8 + 32
+
+Temperature in degrees Fahrenheit is defined as: T(°C) = (T(°F) - 32) / 1.8
+
+## Type conversion
```

Comment:
This heading should be the name of the corresponding task. In general, there are two types of tasks: 

- General hints that apply to all tasks
- Task-specific hints

Where possible, task-specific hints are to be preferred. 

See https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-hints-md for documentation on the `hints.md` file.